aspnetcore/blazor/routing.md

```diff
@@ -102,3 +102,37 @@ There are two `NavLinkMatch` options:
 * `NavLinkMatch.Prefix` &ndash; Specifies that the NavLink should be active when it matches any prefix of the current URL.
 
 In the preceding example, the Home NavLink (`href=""`) matches all URLs and always receives the `active` CSS class. The second NavLink only receives the `active` class when the user visits the Blazor Route component (`href="BlazorRoute"`).
+
+## URI and navigation state helpers
+
+Use `Microsoft.AspNetCore.Components.IUriHelper` to work with URIs and navigation in C# code. `IUriHelper` provides the event and methods shown in the following table.
+
+| Member | Description |
+| ------ | ----------- |
+| `GetAbsoluteUri` | Gets the current absolute URI. |
+| `GetBaseUri` | Gets the base URI (with a trailing slash) that can be prepended to relative URI paths to produce an absolute URI. Typically, this corresponds to the `href` attribute on the document's `<base>` element in *wwwroot/index.html* (Blazor client-side) or *Pages/\_Host.cshtml* (Blazor server-side). |
```

Comment:
"this"  is ambiguous, could refer to base or absolute from previous sentence.

Reply:
It refers back to the member itself. I agree with u ... I probably cut-'n-paste it from the API ... always a dangerous move from an English perspective. :trollface: :smile: lol